simulariumio/cellpack/cellpack_converter.py

```diff
@@ -32,7 +33,12 @@
 
 
 class CellpackConverter(TrajectoryConverter):
-    def __init__(self, input_data: CellpackData):
+    def __init__(
+        self,
+        input_data: CellpackData,
+        progress_callback: Callable = None,
+        num_progress_reports: int = 4,
```

Comment:
is the point of this to limit how many times the progress callback is called?

Reply:
I have no reason for defaulting to 4 honestly, other than the fact that it's an even number. I could default to 10? I wanted to limit it just so we can configure based on different use cases, I am open to suggestions for different default values though

Reply:
10 somehow feels better to me.  It would be cool if we could do them every 3-5 seconds or something. :wink:

Reply:
yeah it would be really cool if it could report percent progress at a given time interval